Tests/test_PDB_MMCIFParser.py

```diff
@@ -278,30 +293,42 @@ def _run_point_mutation_tests(self, parser):
         # Check a non-mutated residue just to be sure we didn't break the
         # parser and cause everyhing to be disordered.
         self.assertFalse(
-            structure[0]["A"][13].is_disordered(),
-            "Residue 13 is not disordered")
+            structure[0]["A"][13].is_disordered(), "Residue 13 is not disordered"
+        )
 
         # Check that the residue types were parsed correctly.
         self.assertSetEqual(
             set(res_1.disordered_get_id_list()),
             {"PRO", "SER"},
-            "Residue 1 is proline/serine")
+            "Residue 1 is proline/serine",
+        )
         self.assertSetEqual(
             set(res_15.disordered_get_id_list()),
             {"ARG", "GLN", "GLU"},
-            "Residue 15 is arginine/glutamine/glutamic acid")
+            "Residue 15 is arginine/glutamine/glutamic acid",
+        )
 
         # Quickly check that we can switch between residues and that the
         # correct set of residues was parsed.
         res_1.disordered_select("PRO")
         self.assertAlmostEqual(
-            res_1["CA"].get_occupancy(),
-            0.83, 2, "Residue 1 proline occupancy correcy")
+            res_1["CA"].get_occupancy(), 0.83, 2, "Residue 1 proline occupancy correcy"
+        )
 
         res_1.disordered_select("SER")
         self.assertAlmostEqual(
-            res_1["CA"].get_occupancy(),
-            0.17, 2, "Residue 1 serine occupancy correcy")
+            res_1["CA"].get_occupancy(), 0.17, 2, "Residue 1 serine occupancy correcy"
+        )
+
+    def test_header(self):
```

Comment:
Perfect. Could you add a 'negative' control test? A mmCIF file without headers to make sure the default values apply correctly?

Reply:
done.

Reply:
Thanks!